README.md

```diff
@@ -65,6 +65,7 @@ Libraries Used
   * [Layout][35] - Lay out widgets using different algorithms.
 * Third party
   * [Glide][90] for image loading
+  * [Kotlin Coroutines][91] for async jobs, plays the role like light-weight threads.
```

Comment:
...[91] for managing background threads with simplified code and reducing needs for callbacks

Reply:
ok